apps/web/styles/globals.css

```diff
@@ -158,6 +162,12 @@ button[role="switch"][data-state="checked"] span {
   .slider .slider-thumb:focus {
     box-shadow: 0 0 0 4px rgba(0, 0, 0, 0.2);
   }
+
+  /* React Select V2 */
+  .cal-react-select-container .cal-react-select__control {
+    @apply bg-white min-h-[36px] h-[36px] text-sm leading-4 border-gray-300 hover:border-neutral-400 focus-within:ring-2 focus-within:ring-neutral-800 focus-within:border-0;
```

Comment:
I believe with this approach we can use `placeholder:text-gray-400 placeholder:text-sm placeholder:font-normal`
to better match the styles when we don't have an item selected

Reply:
Good point. On it!